msteams-platform/concepts/design/design-teams-app-fundamentals.md

```diff
@@ -99,7 +99,7 @@ Quickly learn about the fundamentals of Teams app design. You can find comprehen
 
    :::column-end:::
    :::column span="1":::
-      :::image type="content" source="../../assets/images/design-guidelines/teams-color.png" alt-text="Concept image of Teams colors.":::
+      :::image type="content" source="../../assets/images/design-guidelines/teams-color.png" alt-text="The conceptual image of Teams colors.":::
```

Comment:
Vague alt-text. I don't know what to understand from it when I read `The conceptual image of Teams colors`.
Phrase it better, please.

Reply:
resolved